src/Elastic.Apm/Api/Tracer.cs

```diff
@@ -206,5 +208,74 @@ public Task<T> CaptureTransaction<T>(string name, string type, Func<Task<T>> fun
 
 			transaction.End();
 		}, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
+
+		public void CaptureError(string message, string culprit, StackFrame[] frames = null, string parentId = null,
+			Dictionary<string, Label> labels = null
+		)
+		{
+			var currentTransaction = CurrentExecutionSegmentsContainer.CurrentTransaction;
+
+			IExecutionSegment currentExecutionSegment = CurrentExecutionSegmentsContainer.CurrentSpan;
+			currentExecutionSegment ??= currentTransaction;
+
+			ExecutionSegmentCommon.CaptureError(
+				message,
+				culprit,
+				frames,
+				_sender,
+				_logger,
+				currentExecutionSegment,
+				_configProvider.CurrentSnapshot,
+				currentTransaction as Transaction,
+				_apmServerInfo,
+				parentId,
+				labels
+			);
+		}
+
+		public void CaptureException(Exception exception, string culprit = null, bool isHandled = false, string parentId = null,
+			Dictionary<string, Label> labels = default
+		)
+		{
+			var currentTransaction = CurrentExecutionSegmentsContainer.CurrentTransaction;
+
+			IExecutionSegment currentExecutionSegment = CurrentExecutionSegmentsContainer.CurrentSpan;
+			currentExecutionSegment ??= currentTransaction;
+
+			ExecutionSegmentCommon.CaptureException(
+				exception,
+				_logger,
+				_sender,
+				currentExecutionSegment,
+				_configProvider.CurrentSnapshot,
+				currentTransaction as Transaction,
+				_apmServerInfo,
+				culprit,
+				isHandled,
+				parentId,
+				labels
+			);
+		}
+
+		public void CaptureLogAsError(ErrorLog errorLog, string parentId = null, Exception exception = null, Dictionary<string, Label> labels = null)
+		{
+			var currentTransaction = CurrentExecutionSegmentsContainer.CurrentTransaction;
+
+			IExecutionSegment currentExecutionSegment = CurrentExecutionSegmentsContainer.CurrentSpan;
+			currentExecutionSegment ??= currentTransaction;
```

Comment:
Although it's more verbose, getting the current transaction should be performed only if there's no current span, so that the work is not performed when there is a current span.

Reply:
Same as above.